src/character.cpp

```diff
@@ -2095,7 +2095,7 @@ int Character::i_add_to_container( const item &it, const bool unloading )
 
     const itype_id item_type = it.typeId();
     auto add_to_container = [&it, &charges]( item & container ) {
-        auto &contained_ammo = container.contents.front();
+        item &contained_ammo = container.contents.first_ammo();
```

Comment:
can someone sanity check this line for me? it wasn't actually clear if we were looking only into a magazine, or if "ammo" was just being used as a general word for "an item this will have"